tests/datasets/test_ooddataset.py

```diff
@@ -38,86 +43,138 @@ def test_instanciate_from_tfds():
     assert len(dataset.data.element_spec) == 2
 
 
+def test_instanciate_from_torchvision(erase_after_test=True):
+    temp_root = "./temp_dataset"
+    os.makedirs(temp_root, exist_ok=True)
```

Comment:
I think it can help to use `tempfile.TemporaryDirectory`: https://docs.python.org/3/library/tempfile.html#examples to create temporary folder